### ISSUE TITLE
Only works in Node, not browser

### ISSUE THREAD

**Issue by cliffhall** [krisselden/xoroshiro128starstar#4]
The project requires the fs node package to read the wasm file. Is there any way to compile a version of this that can be used in the browser?

**Comment by krisselden** [comment]
@cliffhall you can use webpack or something like it to make a browser bundle around the wasm module.

**Comment by cliffhall** [comment]
Thanks. I ended up going with [random-js](https://github.com/ckknight/random-js) and its Mersenne Twister implementation. I really would have rather gone with xoroshiro128starstar, but the lift to create a browser module was a bit large for my project's timeframe. 

Perhaps as an addition this project, for anyone would like to contribute and make it more widely usable, add the browserify or webpack config to build a version we can use in a browser. It would be a huge help.

**Comment by krisselden** [comment]
@cliffhall I created this just to have a seedable random that was as fast as Math.random, I could easily make a browser wrapper for it, or like you said use webpack to do the same. It is small enough that the browser should also be able to instantiate synchronously.

**Comment by cliffhall** [comment]
Yeah the webpack route has gotten a lot easier. I built a browser based version of the commonJS domain objects in a project of mine recently and was amazed how easy it was.
